Tools/machines/perlmutter-nersc/perlmutter.sbatch

```diff
@@ -13,22 +13,28 @@
 #SBATCH -A <proj>
 #SBATCH -q regular
 #SBATCH -C gpu
-#SBATCH -c 32
+#SBATCH --exclusive
 #SBATCH --ntasks-per-gpu=1
 #SBATCH --gpus-per-node=4
```

Comment:
@kngott pinged us somewhere (I lost the notification) that this still needs a pinning `none` option to avoid that slurm messes with affinity by using a cgroup...